### ISSUE TITLE
Change of test set

### ISSUE THREAD

**Issue by dinggd** [cmhungsteve/SSTDA#13]
Hi there, great work and thanks for sharing the code. 

I understand that you use the test set as a target domain, does that mean any change of the test set would require retraining to get better results and what are your thoughts/the possible solutions to this?

Thanks and looking forward to your reply.

**Comment by cmhungsteve** [comment]
If you assume that you have no access to the test set (even for unlabeled data) during training, that means you cannot get hyper-parameters from those data. You can try to create a validation set (from training data) to obtain robust hyper-parameters.

**Comment by dinggd** [comment]
Thanks for the reply and I am new to this topic, so can I ask how to get the final results in the paper?

From what I understand is that, for instance, when your train on split1,  you use `split1.train` as the source domain and `split1.test` as the target and then eval on `split1.test`. So naturally, you get the results on `split1` as the default setting in your `eval` code. Similarly, you get results on 4 splits respectively.

So how you do get the overall performance or what's the performance reported in the paper?

Thanks again.

**Comment by cmhungsteve** [comment]
I reported the average results over all splits as others did.
